packages/back-end/src/api/openapi/payload-schemas/PutExperimentPayload.yaml

```diff
@@ -0,0 +1,154 @@
+type: object
+required:
+  - id
+  - name
+  - variations
+  - owner
+  - datasourceId
+  - trackingKey
+  - exposureQueryId
```

Comment:
A couple changes:
- We don't need to require `id` in the payload (it should be required as a URL param)
- None of the top-level fields should be required
  - We should allow partial payloads, however nested objects that have required fields/validations should still be required/enforced (for example, we should keep the `minItems` validation for `variations` field)

Reply:
- Removed `id` from required field and payload schema
- Removed all top-level fields from `required` list